src/pages/DepositBTC.tsx

```diff
@@ -195,11 +211,21 @@ function buildTransactionData(
         .mul(BigNumber.from(10).pow(18))
         .div(estDepositLPTokenAmount.add(poolData?.totalLocked))
     : BigNumber.from(10).pow(18)
+
+  const gasAmount = calculateGasEstimate("addLiquidity").multipliedBy(gasPrice)
+  const txnGasCost = {
+    amount: gasAmount,
+    valueUSD: BigNumber.from(String(TokenPricesUSD.ETH)).multipliedBy(
```

Comment:
Since `TokenPricesUSD.ETH` maybe undefined, let's make this property `null`able and do `TokenPricesUSD.ETH ? BigNumber.from(String(TokenPricesUSD.ETH)).mul(gasAmount) : null`